syft/workers/base.py

```diff
@@ -59,7 +59,13 @@ class BaseWorker(AbstractWorker):
     """
 
     def __init__(
-        self, hook, id=0, data=None, is_client_worker=False, log_msgs=False, verbose=False
+        self,
+        hook: "sy.TorchHook",
```

Comment:
Why is hook a string

Reply:
TorchHook imports BaseWorker. So I can't import hook on BaseWorker. The way to handle this is using [Forward references](https://www.python.org/dev/peps/pep-0484/#forward-references)